next/pages/[...slug].tsx

```diff
@@ -112,5 +112,6 @@ export async function getStaticProps(
         ["common"]
       )),
     },
+    revalidate: 60,
```

Comment:
without this, the page would never be updated when running a production build. From now on, we should always test both with `lando npm run dev` and `lando npm run build && lando npm run start`